test/app/components/inline_component.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class InlineComponent < ViewComponent::Base
+  inline!
```

Comment:
What if we just checked for the presence of a `call` method instead of this explicit flag?

Reply:
That could work but `compile!` method triggers an exception by default when a template file does not exist.

Reply:
What if `compile` checked for the presence of a `call` method?

Reply:
@joelhawksley done, check it out! It also supports the `call_` variants.